packages/datatrak-web/src/features/GroupedSurveyList.tsx

```diff
@@ -21,6 +22,20 @@ const ListWrapper = styled.div`
   }
 `;
 
+type ListItemType = Record<string, unknown> & {
```

Comment:
Thoughts on adding this to `ui-components` too, since it's the same as another component uses?